docs/source/usage.rst

```diff
@@ -59,9 +59,10 @@ Building a specific branch / commit / tag
 =========================================
 
 To build a particular branch and commit, use the argument ``--ref`` and
-specify the ``branch-name`` or ``commit-hash``. For example::
+specify the ``branch-name`` (use the full remote name) or ``commit-hash``. For example::
 
   jupyter-repo2docker --ref 9ced85dd9a84859d0767369e58f33912a214a3cf https://github.com/norvig/pytudes
+  jupyter-repo2docker --ref origin/master https://github.com/norvig/pytudes
```

Comment:
Can we use another named branch here instead while we figure out what we mean here. Also somehting like 

```
or 

jupyter-repo2docker --ref origin/master <commit-hash> https://github.com/norvig/pytudes 
```

Reply:
What do you mean with `jupyter-repo2docker --ref origin/master <commit-hash> ...`? Is it a copy&paste error? Having a branch and a ref isn't something you should specify.

Reply:
Right - the hashes are unique to the branch.  Scrap that then and just say  add the `or` in there